

2024-08-18 20:54:55.872088
20:54:552024-08-18 20:54:55,939 - __main__ - DEBUG - [envy.py:58] - choosing role
2024-08-18 20:54:55,948 - __main__ - DEBUG - [client.py:39] - health checking server
2024-08-18 20:54:55,953 - __main__ - DEBUG - [envy.py:66] - server health check status: True
2024-08-18 20:54:55,954 - __main__ - DEBUG - [envy.py:69] - I am a Client
2024-08-18 20:54:55,954 - __main__ - INFO - [envy.py:128] - Started Envy
2024-08-18 20:54:55,954 - __main__ - DEBUG - [envy.py:149] - envy.connect: Purpose is client
2024-08-18 20:54:55,954 - __main__ - DEBUG - [envy_utils.py:95] - getting server IP
2024-08-18 20:54:55,955 - __main__ - DEBUG - [envy_utils.py:107] - server file found
2024-08-18 20:54:55,956 - __main__ - DEBUG - [envy_utils.py:111] - server ip address is: 10.1.16.179
2024-08-18 20:54:55,956 - __main__ - DEBUG - [client.py:53] - attempting to connect with server at ws://10.1.16.179:37020/client
2024-08-18 20:54:55,959 - __main__ - DEBUG - [client.py:133] - networkUtils.client started
2024-08-18 20:54:55,959 - __main__ - INFO - [client.py:145] - Client: Connected
2024-08-18 20:54:55,959 - __main__ - DEBUG - [client.py:157] - connected to: 10.1.16.179
2024-08-18 20:54:55,959 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 20:54:56,003 - __main__ - INFO - [client.py:162] - received message from: server -> (fill_buffer(self, 'server_name', 'VR1P620-02', ))
2024-08-18 20:54:56,100 - __main__ - INFO - [client.py:162] - received message from: server -> (Task: 75120199573)
2024-08-18 20:54:56,468 - __main__ - DEBUG - [envy.py:257] - executing: await NV.fill_buffer(self, 'server_name', 'VR1P620-02', )
2024-08-18 20:54:56,991 - __main__ - DEBUG - [envy.py:257] - executing: await NV.PLUGIN_eHoudini(self, '{"Purpose": "cache", "ID": 75120199573, "Environment": {"Hip": "blahblah.ma", "Job": "Z:/blahblah"}, "Parameters": {"A": 5}, "Frame": 4}', )
2024-08-18 20:54:57,146 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 20:54:57,147 - __main__ - INFO - [plugin.py:31] - eHoudini: Started process hython_process
2024-08-18 20:56:37,910 - __main__ - INFO - [client.py:162] - received message from: server -> (restart_envy())
2024-08-18 20:56:38,094 - __main__ - DEBUG - [envy.py:257] - executing: await NV.restart_envy(self, )
2024-08-18 20:56:38,213 - __main__ - INFO - [plugin.py:76] - eHoudini: exit function called
2024-08-18 20:56:38,213 - __main__ - INFO - [plugin.py:77] - eHoudini: Terminating child process
2024-08-18 20:56:38,214 - __main__ - INFO - [plugin.py:81] - eHoudini: terminated child process
2024-08-18 20:56:38,214 - __main__ - INFO - [plugin.py:82] - eHoudini: successfully closed


2024-08-18 20:56:39.242212
20:56:392024-08-18 20:56:39,291 - __main__ - DEBUG - [envy.py:58] - choosing role
2024-08-18 20:56:39,299 - __main__ - DEBUG - [client.py:39] - health checking server
2024-08-18 20:56:39,340 - __main__ - DEBUG - [envy.py:66] - server health check status: True
2024-08-18 20:56:39,341 - __main__ - DEBUG - [envy.py:69] - I am a Client
2024-08-18 20:56:39,341 - __main__ - INFO - [envy.py:128] - Started Envy
2024-08-18 20:56:39,341 - __main__ - DEBUG - [envy.py:149] - envy.connect: Purpose is client
2024-08-18 20:56:39,341 - __main__ - DEBUG - [envy_utils.py:95] - getting server IP
2024-08-18 20:56:39,342 - __main__ - DEBUG - [envy_utils.py:107] - server file found
2024-08-18 20:56:39,343 - __main__ - DEBUG - [envy_utils.py:111] - server ip address is: 10.1.16.179
2024-08-18 20:56:39,343 - __main__ - DEBUG - [client.py:53] - attempting to connect with server at ws://10.1.16.179:37020/client
2024-08-18 20:56:39,344 - __main__ - DEBUG - [client.py:133] - networkUtils.client started
2024-08-18 20:56:39,345 - __main__ - INFO - [client.py:145] - Client: Connected
2024-08-18 20:56:39,345 - __main__ - DEBUG - [client.py:157] - connected to: 10.1.16.179
2024-08-18 20:56:39,345 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 20:56:39,401 - __main__ - INFO - [client.py:162] - received message from: server -> (fill_buffer(self, 'server_name', 'VR1P620-02', ))
2024-08-18 20:56:39,841 - __main__ - DEBUG - [envy.py:257] - executing: await NV.fill_buffer(self, 'server_name', 'VR1P620-02', )
2024-08-18 20:56:41,148 - __main__ - DEBUG - [client.py:171] - consumer_handler: no close frame received or sent
2024-08-18 20:56:42,372 - __main__ - WARNING - [client.py:149] - connection closed with server
2024-08-18 20:56:42,372 - __main__ - DEBUG - [client.py:188] - stopping client
2024-08-18 20:56:42,372 - __main__ - DEBUG - [client.py:98] - disconnecting websocket
2024-08-18 20:56:42,372 - __main__ - DEBUG - [client.py:106] - Connection Closed Successfully
2024-08-18 20:56:42,373 - __main__ - DEBUG - [client.py:199] - cleaning up tasks
2024-08-18 20:56:42,373 - __main__ - DEBUG - [client.py:202] - cancelling task client.consumer_handler
2024-08-18 20:56:42,373 - __main__ - DEBUG - [client.py:212] - client stopped
2024-08-18 20:56:42,373 - __main__ - DEBUG - [envy.py:198] - cleaning up envy.client_dependant_tasks
2024-08-18 20:56:42,373 - __main__ - INFO - [envy.py:214] - electing new server
2024-08-18 20:56:42,373 - __main__ - DEBUG - [envy_utils.py:149] - opening clients file at Z:/school/ENVY/__ENVYTEST__/Connections/clients.json
2024-08-18 20:56:42,381 - __main__ - DEBUG - [envy_utils.py:154] - returning clients: {'VR1P620-04': {'IP': '10.1.16.181', 'Status': 'idle', 'Job': None, 'Task': None, 'Progress': 0}, 'VR1P620-02': {'IP': '10.1.16.179', 'Status': 'idle', 'Job': None, 'Task': None, 'Progress': 0}, 'VR1P620-05': {'IP': '10.1.16.182', 'Status': 'idle', 'Job': None, 'Task': None, 'Progress': 0}}
2024-08-18 20:56:42,381 - __main__ - DEBUG - [envy.py:222] - removing old server from eligible clients
2024-08-18 20:56:42,382 - __main__ - DEBUG - [envy.py:228] - New server: VR1P620-04
2024-08-18 20:56:47,382 - __main__ - DEBUG - [envy.py:149] - envy.connect: Purpose is client
2024-08-18 20:56:47,382 - __main__ - DEBUG - [envy_utils.py:95] - getting server IP
2024-08-18 20:56:47,384 - __main__ - DEBUG - [envy_utils.py:107] - server file found
2024-08-18 20:56:47,394 - __main__ - DEBUG - [envy_utils.py:111] - server ip address is: 10.1.16.181
2024-08-18 20:56:47,394 - __main__ - DEBUG - [client.py:53] - attempting to connect with server at ws://10.1.16.181:37020/client
2024-08-18 20:56:47,395 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 20:56:47,396 - __main__ - DEBUG - [client.py:133] - networkUtils.client started
2024-08-18 20:56:47,396 - __main__ - INFO - [client.py:145] - Client: Connected
2024-08-18 20:56:47,397 - __main__ - DEBUG - [client.py:157] - connected to: 10.1.16.181
2024-08-18 20:56:47,441 - __main__ - INFO - [client.py:162] - received message from: server -> (fill_buffer(self, 'server_name', 'VR1P620-04', ))
2024-08-18 20:56:47,913 - __main__ - DEBUG - [envy.py:257] - executing: await NV.fill_buffer(self, 'server_name', 'VR1P620-04', )
2024-08-18 20:56:48,906 - __main__ - INFO - [client.py:162] - received message from: server -> (Task: 75120199572)
2024-08-18 20:56:48,934 - __main__ - DEBUG - [envy.py:257] - executing: await NV.PLUGIN_eHoudini(self, '{"Purpose": "cache", "ID": 75120199572, "Environment": {"Hip": "blahblah.ma", "Job": "Z:/blahblah"}, "Parameters": {"A": 5}, "Frame": 3}', )
2024-08-18 20:56:49,014 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 20:56:49,015 - __main__ - INFO - [plugin.py:31] - eHoudini: Started process hython_process
2024-08-18 20:57:11,382 - __main__ - INFO - [client.py:162] - received message from: server -> (restart_envy())
2024-08-18 20:57:11,703 - __main__ - DEBUG - [envy.py:257] - executing: await NV.restart_envy(self, )
2024-08-18 20:57:11,805 - __main__ - INFO - [plugin.py:76] - eHoudini: exit function called
2024-08-18 20:57:11,806 - __main__ - INFO - [plugin.py:77] - eHoudini: Terminating child process
2024-08-18 20:57:11,806 - __main__ - INFO - [plugin.py:81] - eHoudini: terminated child process
2024-08-18 20:57:11,807 - __main__ - INFO - [plugin.py:82] - eHoudini: successfully closed


2024-08-18 20:57:12.640624
20:57:122024-08-18 20:57:12,687 - __main__ - DEBUG - [envy.py:58] - choosing role
2024-08-18 20:57:12,693 - __main__ - DEBUG - [client.py:39] - health checking server
2024-08-18 20:57:12,702 - __main__ - DEBUG - [envy.py:66] - server health check status: True
2024-08-18 20:57:12,702 - __main__ - DEBUG - [envy.py:69] - I am a Client
2024-08-18 20:57:12,703 - __main__ - INFO - [envy.py:128] - Started Envy
2024-08-18 20:57:12,703 - __main__ - DEBUG - [envy.py:149] - envy.connect: Purpose is client
2024-08-18 20:57:12,703 - __main__ - DEBUG - [envy_utils.py:95] - getting server IP
2024-08-18 20:57:12,704 - __main__ - DEBUG - [envy_utils.py:107] - server file found
2024-08-18 20:57:12,705 - __main__ - DEBUG - [envy_utils.py:111] - server ip address is: 10.1.16.181
2024-08-18 20:57:12,705 - __main__ - DEBUG - [client.py:53] - attempting to connect with server at ws://10.1.16.181:37020/client
2024-08-18 20:57:12,707 - __main__ - DEBUG - [client.py:133] - networkUtils.client started
2024-08-18 20:57:12,707 - __main__ - INFO - [client.py:145] - Client: Connected
2024-08-18 20:57:12,707 - __main__ - DEBUG - [client.py:157] - connected to: 10.1.16.181
2024-08-18 20:57:12,707 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 20:57:12,720 - __main__ - INFO - [client.py:162] - received message from: server -> (fill_buffer(self, 'server_name', 'VR1P620-04', ))
2024-08-18 20:57:13,202 - __main__ - DEBUG - [envy.py:257] - executing: await NV.fill_buffer(self, 'server_name', 'VR1P620-04', )
2024-08-18 20:57:13,840 - __main__ - DEBUG - [client.py:171] - consumer_handler: no close frame received or sent
2024-08-18 20:57:14,206 - __main__ - WARNING - [client.py:149] - connection closed with server
2024-08-18 20:57:14,206 - __main__ - DEBUG - [client.py:188] - stopping client
2024-08-18 20:57:14,206 - __main__ - DEBUG - [client.py:98] - disconnecting websocket
2024-08-18 20:57:14,206 - __main__ - DEBUG - [client.py:106] - Connection Closed Successfully
2024-08-18 20:57:14,207 - __main__ - DEBUG - [client.py:199] - cleaning up tasks
2024-08-18 20:57:14,207 - __main__ - DEBUG - [client.py:202] - cancelling task client.consumer_handler
2024-08-18 20:57:14,207 - __main__ - DEBUG - [client.py:212] - client stopped
2024-08-18 20:57:14,207 - __main__ - DEBUG - [envy.py:198] - cleaning up envy.client_dependant_tasks
2024-08-18 20:57:14,207 - __main__ - INFO - [envy.py:214] - electing new server
2024-08-18 20:57:14,207 - __main__ - DEBUG - [envy_utils.py:149] - opening clients file at Z:/school/ENVY/__ENVYTEST__/Connections/clients.json
2024-08-18 20:57:14,215 - __main__ - DEBUG - [envy_utils.py:154] - returning clients: {'VR1P620-04': {'IP': '10.1.16.181', 'Status': 'idle', 'Job': None, 'Task': None, 'Progress': 0}, 'VR1P620-05': {'IP': '10.1.16.182', 'Status': 'idle', 'Job': None, 'Task': None, 'Progress': 0}, 'VR1P620-02': {'IP': '10.1.16.179', 'Status': 'idle', 'Job': None, 'Task': None, 'Progress': 0}}
2024-08-18 20:57:14,215 - __main__ - DEBUG - [envy.py:222] - removing old server from eligible clients
2024-08-18 20:57:14,216 - __main__ - DEBUG - [envy.py:228] - New server: VR1P620-02
2024-08-18 20:57:19,217 - __main__ - DEBUG - [envy.py:149] - envy.connect: Purpose is client
2024-08-18 20:57:19,217 - __main__ - DEBUG - [envy_utils.py:95] - getting server IP
2024-08-18 20:57:19,219 - __main__ - DEBUG - [envy_utils.py:107] - server file found
2024-08-18 20:57:19,226 - __main__ - DEBUG - [envy_utils.py:111] - server ip address is: 10.1.16.181
2024-08-18 20:57:19,227 - __main__ - DEBUG - [client.py:53] - attempting to connect with server at ws://10.1.16.181:37020/client
2024-08-18 20:57:19,227 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 20:57:19,229 - __main__ - DEBUG - [client.py:133] - networkUtils.client started
2024-08-18 20:57:19,229 - __main__ - INFO - [client.py:145] - Client: Connected
2024-08-18 20:57:19,230 - __main__ - DEBUG - [client.py:157] - connected to: 10.1.16.181
2024-08-18 20:57:19,284 - __main__ - INFO - [client.py:162] - received message from: server -> (fill_buffer(self, 'server_name', 'VR1P620-04', ))
2024-08-18 20:57:19,742 - __main__ - DEBUG - [envy.py:257] - executing: await NV.fill_buffer(self, 'server_name', 'VR1P620-04', )
2024-08-18 20:57:20,963 - __main__ - INFO - [client.py:162] - received message from: server -> (Task: 75120199574)
2024-08-18 20:57:21,269 - __main__ - DEBUG - [envy.py:257] - executing: await NV.PLUGIN_eHoudini(self, '{"Purpose": "cache", "ID": 75120199574, "Environment": {"Hip": "blahblah.ma", "Job": "Z:/blahblah"}, "Parameters": {"A": 5}, "Frame": 5}', )
2024-08-18 20:57:21,346 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 20:57:21,347 - __main__ - INFO - [plugin.py:31] - eHoudini: Started process hython_process
2024-08-18 20:57:55,614 - __main__ - DEBUG - [client.py:171] - consumer_handler: no close frame received or sent
2024-08-18 20:57:56,958 - __main__ - WARNING - [client.py:149] - connection closed with server
2024-08-18 20:57:56,958 - __main__ - DEBUG - [client.py:188] - stopping client
2024-08-18 20:57:56,959 - __main__ - DEBUG - [client.py:98] - disconnecting websocket
2024-08-18 20:57:56,959 - __main__ - DEBUG - [client.py:106] - Connection Closed Successfully
2024-08-18 20:57:56,959 - __main__ - DEBUG - [client.py:199] - cleaning up tasks
2024-08-18 20:57:56,960 - __main__ - DEBUG - [client.py:202] - cancelling task client.producer
2024-08-18 20:57:56,960 - __main__ - DEBUG - [client.py:202] - cancelling task client.producer
2024-08-18 20:57:56,960 - __main__ - DEBUG - [client.py:210] - cancelled task client.producer
2024-08-18 20:57:56,961 - __main__ - DEBUG - [client.py:212] - client stopped
2024-08-18 20:57:56,961 - __main__ - DEBUG - [envy.py:198] - cleaning up envy.client_dependant_tasks
2024-08-18 20:57:56,961 - __main__ - INFO - [envy.py:214] - electing new server
2024-08-18 20:57:56,961 - __main__ - DEBUG - [envy_utils.py:149] - opening clients file at Z:/school/ENVY/__ENVYTEST__/Connections/clients.json
2024-08-18 20:57:56,969 - __main__ - DEBUG - [envy_utils.py:154] - returning clients: {'VR1P620-02': {'IP': '10.1.16.179', 'Status': 'working', 'Job': 7512019957, 'Task': 75120199572, 'Progress': 0}, 'VR1P620-05': {'IP': '10.1.16.182', 'Status': 'working', 'Job': 7512019957, 'Task': 75120199574, 'Progress': 0}}
2024-08-18 20:57:56,969 - __main__ - DEBUG - [envy.py:228] - New server: VR1P620-02
2024-08-18 20:58:01,970 - __main__ - DEBUG - [envy.py:149] - envy.connect: Purpose is client
2024-08-18 20:58:01,970 - __main__ - DEBUG - [envy_utils.py:95] - getting server IP
2024-08-18 20:58:01,973 - __main__ - DEBUG - [envy_utils.py:107] - server file found
2024-08-18 20:58:01,983 - __main__ - DEBUG - [envy_utils.py:111] - server ip address is: 10.1.16.179
2024-08-18 20:58:01,983 - __main__ - DEBUG - [client.py:53] - attempting to connect with server at ws://10.1.16.179:37020/client
2024-08-18 20:58:01,986 - __main__ - DEBUG - [client.py:133] - networkUtils.client started
2024-08-18 20:58:01,986 - __main__ - INFO - [client.py:145] - Client: Connected
2024-08-18 20:58:01,986 - __main__ - DEBUG - [client.py:157] - connected to: 10.1.16.179
2024-08-18 20:58:01,986 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 20:58:02,004 - __main__ - INFO - [client.py:162] - received message from: server -> (fill_buffer(self, 'server_name', 'VR1P620-02', ))
2024-08-18 20:58:02,484 - __main__ - DEBUG - [envy.py:257] - executing: await NV.fill_buffer(self, 'server_name', 'VR1P620-02', )
2024-08-18 20:58:41,759 - __main__ - INFO - [client.py:162] - received message from: server -> (EXAMPLE())
2024-08-18 20:58:42,195 - __main__ - DEBUG - [envy.py:257] - executing: await NV.ENVY_EXAMPLE(self, 10, arg2=''potato'')
2024-08-18 20:58:42,195 - __main__ - ERROR - [envy.py:85] - Failed while executing ENVY_EXAMPLE(self, 10, arg2=''potato''), invalid syntax. Perhaps you forgot a comma? (<string>, line 1)
2024-08-18 20:59:04,151 - __main__ - INFO - [client.py:162] - received message from: server -> (EXAMPLE())
2024-08-18 20:59:04,507 - __main__ - DEBUG - [envy.py:257] - executing: await NV.ENVY_EXAMPLE(self, 50, arg2='potato')
2024-08-18 20:59:51,223 - __main__ - INFO - [client.py:162] - received message from: server -> (restart_envy())
2024-08-18 20:59:51,630 - __main__ - DEBUG - [envy.py:257] - executing: await NV.restart_envy(self, )
2024-08-18 20:59:51,746 - __main__ - INFO - [plugin.py:76] - eHoudini: exit function called
2024-08-18 20:59:51,747 - __main__ - INFO - [plugin.py:77] - eHoudini: Terminating child process
2024-08-18 20:59:51,747 - __main__ - INFO - [plugin.py:81] - eHoudini: terminated child process
2024-08-18 20:59:51,748 - __main__ - INFO - [plugin.py:82] - eHoudini: successfully closed


2024-08-18 20:59:52.670103
20:59:522024-08-18 20:59:52,717 - __main__ - DEBUG - [envy.py:58] - choosing role
2024-08-18 20:59:52,724 - __main__ - DEBUG - [client.py:39] - health checking server
2024-08-18 20:59:52,729 - __main__ - DEBUG - [envy.py:66] - server health check status: True
2024-08-18 20:59:52,729 - __main__ - DEBUG - [envy.py:69] - I am a Client
2024-08-18 20:59:52,729 - __main__ - INFO - [envy.py:128] - Started Envy
2024-08-18 20:59:52,729 - __main__ - DEBUG - [envy.py:149] - envy.connect: Purpose is client
2024-08-18 20:59:52,729 - __main__ - DEBUG - [envy_utils.py:95] - getting server IP
2024-08-18 20:59:52,730 - __main__ - DEBUG - [envy_utils.py:107] - server file found
2024-08-18 20:59:52,731 - __main__ - DEBUG - [envy_utils.py:111] - server ip address is: 10.1.16.179
2024-08-18 20:59:52,731 - __main__ - DEBUG - [client.py:53] - attempting to connect with server at ws://10.1.16.179:37020/client
2024-08-18 20:59:52,733 - __main__ - DEBUG - [client.py:133] - networkUtils.client started
2024-08-18 20:59:52,733 - __main__ - INFO - [client.py:145] - Client: Connected
2024-08-18 20:59:52,733 - __main__ - DEBUG - [client.py:157] - connected to: 10.1.16.179
2024-08-18 20:59:52,733 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 20:59:52,777 - __main__ - INFO - [client.py:162] - received message from: server -> (fill_buffer(self, 'server_name', 'VR1P620-02', ))
2024-08-18 20:59:53,252 - __main__ - DEBUG - [envy.py:257] - executing: await NV.fill_buffer(self, 'server_name', 'VR1P620-02', )
2024-08-18 20:59:53,290 - __main__ - DEBUG - [client.py:171] - consumer_handler: no close frame received or sent
2024-08-18 20:59:54,259 - __main__ - WARNING - [client.py:149] - connection closed with server
2024-08-18 20:59:54,259 - __main__ - DEBUG - [client.py:188] - stopping client
2024-08-18 20:59:54,259 - __main__ - DEBUG - [client.py:98] - disconnecting websocket
2024-08-18 20:59:54,259 - __main__ - DEBUG - [client.py:106] - Connection Closed Successfully
2024-08-18 20:59:54,260 - __main__ - DEBUG - [client.py:199] - cleaning up tasks
2024-08-18 20:59:54,260 - __main__ - DEBUG - [client.py:202] - cancelling task client.consumer_handler
2024-08-18 20:59:54,260 - __main__ - DEBUG - [client.py:212] - client stopped
2024-08-18 20:59:54,260 - __main__ - DEBUG - [envy.py:198] - cleaning up envy.client_dependant_tasks
2024-08-18 20:59:54,260 - __main__ - INFO - [envy.py:214] - electing new server
2024-08-18 20:59:54,260 - __main__ - DEBUG - [envy_utils.py:149] - opening clients file at Z:/school/ENVY/__ENVYTEST__/Connections/clients.json
2024-08-18 20:59:54,268 - __main__ - DEBUG - [envy_utils.py:154] - returning clients: {'VR1P620-02': {'IP': '10.1.16.179', 'Status': 'idle', 'Job': None, 'Task': None, 'Progress': 0}, 'VR1P620-05': {'IP': '10.1.16.182', 'Status': 'idle', 'Job': None, 'Task': None, 'Progress': 0}}
2024-08-18 20:59:54,268 - __main__ - DEBUG - [envy.py:222] - removing old server from eligible clients
2024-08-18 20:59:54,268 - __main__ - DEBUG - [envy.py:228] - New server: VR1P620-05
2024-08-18 20:59:54,269 - __main__ - DEBUG - [envy.py:149] - envy.connect: Purpose is server
2024-08-18 20:59:54,269 - __main__ - DEBUG - [envy.py:106] - writing server file
2024-08-18 20:59:54,287 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 20:59:55,296 - __main__ - DEBUG - [envy_utils.py:95] - getting server IP
2024-08-18 20:59:55,297 - __main__ - DEBUG - [envy_utils.py:107] - server file found
2024-08-18 20:59:55,304 - __main__ - DEBUG - [envy_utils.py:111] - server ip address is: 10.1.16.179
2024-08-18 20:59:55,304 - __main__ - DEBUG - [client.py:53] - attempting to connect with server at ws://10.1.16.179:37020/client
2024-08-18 20:59:55,306 - __main__ - DEBUG - [client.py:133] - networkUtils.client started
2024-08-18 20:59:55,306 - __main__ - INFO - [client.py:145] - Client: Connected
2024-08-18 20:59:55,307 - __main__ - DEBUG - [client.py:157] - connected to: 10.1.16.179
2024-08-18 20:59:55,359 - __main__ - INFO - [client.py:162] - received message from: server -> (fill_buffer(self, 'server_name', 'VR1P620-02', ))
2024-08-18 20:59:55,801 - __main__ - DEBUG - [envy.py:257] - executing: await NV.fill_buffer(self, 'server_name', 'VR1P620-02', )
2024-08-18 20:59:56,553 - __main__ - INFO - [client.py:162] - received message from: server -> (Task: 75120199572)
2024-08-18 20:59:56,812 - __main__ - DEBUG - [envy.py:257] - executing: await NV.PLUGIN_eHoudini(self, '{"Purpose": "cache", "ID": 75120199572, "Environment": {"Hip": "blahblah.ma", "Job": "Z:/blahblah"}, "Parameters": {"A": 5}, "Frame": 3}', )
2024-08-18 20:59:56,869 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 20:59:56,869 - __main__ - INFO - [plugin.py:31] - eHoudini: Started process hython_process
2024-08-18 21:03:56,495 - __main__ - INFO - [client.py:162] - received message from: server -> (restart_envy())
2024-08-18 21:03:56,893 - __main__ - DEBUG - [envy.py:257] - executing: await NV.restart_envy(self, )
2024-08-18 21:03:57,001 - __main__ - INFO - [plugin.py:76] - eHoudini: exit function called
2024-08-18 21:03:57,001 - __main__ - INFO - [plugin.py:77] - eHoudini: Terminating child process
2024-08-18 21:03:57,002 - __main__ - INFO - [plugin.py:81] - eHoudini: terminated child process
2024-08-18 21:03:57,002 - __main__ - INFO - [plugin.py:82] - eHoudini: successfully closed


2024-08-18 21:03:57.964909
21:03:572024-08-18 21:03:58,014 - __main__ - DEBUG - [envy.py:58] - choosing role
2024-08-18 21:03:58,023 - __main__ - DEBUG - [client.py:39] - health checking server
2024-08-18 21:03:58,028 - __main__ - DEBUG - [envy.py:66] - server health check status: True
2024-08-18 21:03:58,028 - __main__ - DEBUG - [envy.py:69] - I am a Client
2024-08-18 21:03:58,029 - __main__ - INFO - [envy.py:128] - Started Envy
2024-08-18 21:03:58,029 - __main__ - DEBUG - [envy.py:149] - envy.connect: Purpose is client
2024-08-18 21:03:58,029 - __main__ - DEBUG - [envy_utils.py:95] - getting server IP
2024-08-18 21:03:58,029 - __main__ - DEBUG - [envy_utils.py:107] - server file found
2024-08-18 21:03:58,031 - __main__ - DEBUG - [envy_utils.py:111] - server ip address is: 10.1.16.179
2024-08-18 21:03:58,031 - __main__ - DEBUG - [client.py:53] - attempting to connect with server at ws://10.1.16.179:37020/client
2024-08-18 21:03:58,033 - __main__ - DEBUG - [client.py:133] - networkUtils.client started
2024-08-18 21:03:58,033 - __main__ - INFO - [client.py:145] - Client: Connected
2024-08-18 21:03:58,033 - __main__ - DEBUG - [client.py:157] - connected to: 10.1.16.179
2024-08-18 21:03:58,033 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:03:58,048 - __main__ - INFO - [client.py:162] - received message from: server -> (fill_buffer(self, 'server_name', 'VR1P620-02', ))
2024-08-18 21:03:58,534 - __main__ - DEBUG - [envy.py:257] - executing: await NV.fill_buffer(self, 'server_name', 'VR1P620-02', )
2024-08-18 21:03:58,943 - __main__ - DEBUG - [client.py:171] - consumer_handler: no close frame received or sent
2024-08-18 21:03:59,534 - __main__ - WARNING - [client.py:149] - connection closed with server
2024-08-18 21:03:59,534 - __main__ - DEBUG - [client.py:188] - stopping client
2024-08-18 21:03:59,534 - __main__ - DEBUG - [client.py:98] - disconnecting websocket
2024-08-18 21:03:59,535 - __main__ - DEBUG - [client.py:106] - Connection Closed Successfully
2024-08-18 21:03:59,535 - __main__ - DEBUG - [client.py:199] - cleaning up tasks
2024-08-18 21:03:59,535 - __main__ - DEBUG - [client.py:202] - cancelling task client.consumer_handler
2024-08-18 21:03:59,535 - __main__ - DEBUG - [client.py:212] - client stopped
2024-08-18 21:03:59,535 - __main__ - DEBUG - [envy.py:198] - cleaning up envy.client_dependant_tasks
2024-08-18 21:03:59,535 - __main__ - INFO - [envy.py:214] - electing new server
2024-08-18 21:03:59,535 - __main__ - DEBUG - [envy_utils.py:149] - opening clients file at Z:/school/ENVY/__ENVYTEST__/Connections/clients.json
2024-08-18 21:03:59,550 - __main__ - DEBUG - [envy_utils.py:154] - returning clients: {'VR1P620-05': {'IP': '10.1.16.182', 'Status': 'idle', 'Job': None, 'Task': None, 'Progress': 0}, 'VR1P620-02': {'IP': '10.1.16.179', 'Status': 'idle', 'Job': None, 'Task': None, 'Progress': 0}}
2024-08-18 21:03:59,550 - __main__ - DEBUG - [envy.py:222] - removing old server from eligible clients
2024-08-18 21:03:59,550 - __main__ - DEBUG - [envy.py:228] - New server: VR1P620-05
2024-08-18 21:03:59,551 - __main__ - DEBUG - [envy.py:149] - envy.connect: Purpose is server
2024-08-18 21:03:59,551 - __main__ - DEBUG - [envy.py:106] - writing server file
2024-08-18 21:03:59,575 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:04:00,589 - __main__ - DEBUG - [envy_utils.py:95] - getting server IP
2024-08-18 21:04:00,590 - __main__ - DEBUG - [envy_utils.py:107] - server file found
2024-08-18 21:04:00,597 - __main__ - DEBUG - [envy_utils.py:111] - server ip address is: 10.1.16.182
2024-08-18 21:04:00,597 - __main__ - DEBUG - [client.py:53] - attempting to connect with server at ws://10.1.16.182:37020/client
2024-08-18 21:04:00,599 - __main__ - DEBUG - [client.py:133] - networkUtils.client started
2024-08-18 21:04:00,599 - __main__ - INFO - [client.py:145] - Client: Connected
2024-08-18 21:04:00,600 - __main__ - DEBUG - [client.py:157] - connected to: 10.1.16.182
2024-08-18 21:04:00,606 - __main__ - INFO - [client.py:162] - received message from: server -> (fill_buffer(self, 'server_name', 'VR1P620-05', ))
2024-08-18 21:04:01,045 - __main__ - DEBUG - [envy.py:257] - executing: await NV.fill_buffer(self, 'server_name', 'VR1P620-05', )
2024-08-18 21:04:02,289 - __main__ - INFO - [client.py:162] - received message from: server -> (Task: 75120199570)
2024-08-18 21:04:02,585 - __main__ - DEBUG - [envy.py:257] - executing: await NV.PLUGIN_eHoudini(self, '{"Purpose": "cache", "ID": 75120199570, "Environment": {"Hip": "blahblah.ma", "Job": "Z:/blahblah"}, "Parameters": {"A": 5}, "Frame": 1}', )
2024-08-18 21:04:02,639 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:04:02,639 - __main__ - INFO - [plugin.py:31] - eHoudini: Started process hython_process
2024-08-18 21:04:08,390 - __main__ - INFO - [client.py:162] - received message from: server -> (shit())
2024-08-18 21:04:08,716 - __main__ - DEBUG - [envy.py:257] - executing: await NV.shit(self, )
2024-08-18 21:04:16,910 - __main__ - INFO - [client.py:162] - received message from: server -> (restart_envy())


2024-08-18 21:05:02.074941
21:05:022024-08-18 21:05:02,123 - __main__ - DEBUG - [envy.py:58] - choosing role
2024-08-18 21:05:02,133 - __main__ - DEBUG - [client.py:39] - health checking server
2024-08-18 21:05:04,179 - __main__ - DEBUG - [envy.py:66] - server health check status: False
2024-08-18 21:05:04,182 - __main__ - DEBUG - [envy.py:75] - I am the server
2024-08-18 21:05:04,182 - __main__ - INFO - [envy.py:128] - Started Envy
2024-08-18 21:05:04,182 - __main__ - DEBUG - [envy.py:149] - envy.connect: Purpose is server
2024-08-18 21:05:04,183 - __main__ - DEBUG - [envy.py:106] - writing server file
2024-08-18 21:05:05,227 - __main__ - DEBUG - [envy_utils.py:95] - getting server IP
2024-08-18 21:05:05,228 - __main__ - DEBUG - [envy_utils.py:107] - server file found
2024-08-18 21:05:05,234 - __main__ - DEBUG - [envy_utils.py:111] - server ip address is: 10.1.16.182
2024-08-18 21:05:05,234 - __main__ - DEBUG - [client.py:53] - attempting to connect with server at ws://10.1.16.182:37020/client
2024-08-18 21:05:05,236 - __main__ - DEBUG - [client.py:133] - networkUtils.client started
2024-08-18 21:05:05,236 - __main__ - INFO - [client.py:145] - Client: Connected
2024-08-18 21:05:05,237 - __main__ - DEBUG - [client.py:157] - connected to: 10.1.16.182
2024-08-18 21:05:05,237 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:05:05,249 - __main__ - INFO - [client.py:162] - received message from: server -> (fill_buffer(self, 'server_name', 'VR1P620-05', ))
2024-08-18 21:05:05,743 - __main__ - DEBUG - [envy.py:257] - executing: await NV.fill_buffer(self, 'server_name', 'VR1P620-05', )
2024-08-18 21:05:06,874 - __main__ - INFO - [client.py:162] - received message from: server -> (Task: 75120199570)
2024-08-18 21:05:07,257 - __main__ - DEBUG - [envy.py:257] - executing: await NV.PLUGIN_eHoudini(self, '{"Purpose": "cache", "ID": 75120199570, "Environment": {"Hip": "blahblah.ma", "Job": "Z:/blahblah"}, "Parameters": {"A": 5}, "Frame": 1}', )
2024-08-18 21:05:07,313 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:05:07,314 - __main__ - INFO - [plugin.py:31] - eHoudini: Started process hython_process


2024-08-18 21:05:56.524030
21:05:562024-08-18 21:05:56,572 - __main__ - DEBUG - [envy.py:58] - choosing role
2024-08-18 21:05:56,579 - __main__ - DEBUG - [client.py:39] - health checking server
2024-08-18 21:05:58,612 - __main__ - DEBUG - [envy.py:66] - server health check status: False
2024-08-18 21:05:58,615 - __main__ - DEBUG - [envy.py:75] - I am the server
2024-08-18 21:05:58,615 - __main__ - INFO - [envy.py:128] - Started Envy
2024-08-18 21:05:58,615 - __main__ - DEBUG - [envy.py:149] - envy.connect: Purpose is server
2024-08-18 21:05:58,615 - __main__ - DEBUG - [envy.py:106] - writing server file
2024-08-18 21:05:59,629 - __main__ - DEBUG - [envy_utils.py:95] - getting server IP
2024-08-18 21:05:59,630 - __main__ - DEBUG - [envy_utils.py:107] - server file found
2024-08-18 21:05:59,636 - __main__ - DEBUG - [envy_utils.py:111] - server ip address is: 10.1.16.182
2024-08-18 21:05:59,636 - __main__ - DEBUG - [client.py:53] - attempting to connect with server at ws://10.1.16.182:37020/client
2024-08-18 21:05:59,639 - __main__ - DEBUG - [client.py:133] - networkUtils.client started
2024-08-18 21:05:59,639 - __main__ - INFO - [client.py:145] - Client: Connected
2024-08-18 21:05:59,639 - __main__ - DEBUG - [client.py:157] - connected to: 10.1.16.182
2024-08-18 21:05:59,640 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:05:59,652 - __main__ - INFO - [client.py:162] - received message from: server -> (fill_buffer(self, 'server_name', 'VR1P620-05', ))
2024-08-18 21:06:00,161 - __main__ - DEBUG - [envy.py:257] - executing: await NV.fill_buffer(self, 'server_name', 'VR1P620-05', )
2024-08-18 21:07:21,183 - __main__ - INFO - [client.py:162] - received message from: server -> (restart_envy())
2024-08-18 21:07:21,231 - __main__ - DEBUG - [envy.py:257] - executing: await NV.restart_envy(self, )


2024-08-18 21:07:22.398283
21:07:222024-08-18 21:07:22,444 - __main__ - DEBUG - [envy.py:58] - choosing role
2024-08-18 21:07:22,451 - __main__ - DEBUG - [client.py:39] - health checking server
2024-08-18 21:07:22,456 - __main__ - DEBUG - [envy.py:66] - server health check status: True
2024-08-18 21:07:22,456 - __main__ - DEBUG - [envy.py:69] - I am a Client
2024-08-18 21:07:22,457 - __main__ - INFO - [envy.py:128] - Started Envy
2024-08-18 21:07:22,457 - __main__ - DEBUG - [envy.py:149] - envy.connect: Purpose is client
2024-08-18 21:07:22,457 - __main__ - DEBUG - [envy_utils.py:95] - getting server IP
2024-08-18 21:07:22,458 - __main__ - DEBUG - [envy_utils.py:107] - server file found
2024-08-18 21:07:22,460 - __main__ - DEBUG - [envy_utils.py:111] - server ip address is: 10.1.16.182
2024-08-18 21:07:22,460 - __main__ - DEBUG - [client.py:53] - attempting to connect with server at ws://10.1.16.182:37020/client
2024-08-18 21:07:22,461 - __main__ - DEBUG - [client.py:133] - networkUtils.client started
2024-08-18 21:07:22,461 - __main__ - INFO - [client.py:145] - Client: Connected
2024-08-18 21:07:22,462 - __main__ - DEBUG - [client.py:157] - connected to: 10.1.16.182
2024-08-18 21:07:22,462 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:07:22,477 - __main__ - INFO - [client.py:162] - received message from: server -> (fill_buffer(self, 'server_name', 'VR1P620-05', ))
2024-08-18 21:07:22,956 - __main__ - DEBUG - [envy.py:257] - executing: await NV.fill_buffer(self, 'server_name', 'VR1P620-05', )
2024-08-18 21:07:23,284 - __main__ - DEBUG - [client.py:171] - consumer_handler: no close frame received or sent
2024-08-18 21:07:23,957 - __main__ - WARNING - [client.py:149] - connection closed with server
2024-08-18 21:07:23,957 - __main__ - DEBUG - [client.py:188] - stopping client
2024-08-18 21:07:23,957 - __main__ - DEBUG - [client.py:98] - disconnecting websocket
2024-08-18 21:07:23,957 - __main__ - DEBUG - [client.py:106] - Connection Closed Successfully
2024-08-18 21:07:23,957 - __main__ - DEBUG - [client.py:199] - cleaning up tasks
2024-08-18 21:07:23,957 - __main__ - DEBUG - [client.py:202] - cancelling task client.consumer_handler
2024-08-18 21:07:23,958 - __main__ - DEBUG - [client.py:212] - client stopped
2024-08-18 21:07:23,958 - __main__ - DEBUG - [envy.py:198] - cleaning up envy.client_dependant_tasks
2024-08-18 21:07:23,958 - __main__ - INFO - [envy.py:214] - electing new server
2024-08-18 21:07:23,958 - __main__ - DEBUG - [envy_utils.py:149] - opening clients file at Z:/school/ENVY/__ENVYTEST__/Connections/clients.json
2024-08-18 21:07:23,968 - __main__ - DEBUG - [envy_utils.py:154] - returning clients: {'VR1P620-05': {'IP': '10.1.16.182', 'Status': 'idle', 'Job': None, 'Task': None, 'Progress': 0}, 'VR1P620-02': {'IP': '10.1.16.179', 'Status': 'idle', 'Job': None, 'Task': None, 'Progress': 0}, 'VR1P620-04': {'IP': '10.1.16.181', 'Status': 'idle', 'Job': None, 'Task': None, 'Progress': 0}}
2024-08-18 21:07:23,969 - __main__ - DEBUG - [envy.py:149] - envy.connect: Purpose is server
2024-08-18 21:07:23,969 - __main__ - DEBUG - [envy.py:106] - writing server file
2024-08-18 21:07:24,005 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:07:24,996 - __main__ - DEBUG - [envy_utils.py:95] - getting server IP
2024-08-18 21:07:24,997 - __main__ - DEBUG - [envy_utils.py:107] - server file found
2024-08-18 21:07:25,003 - __main__ - DEBUG - [envy_utils.py:111] - server ip address is: 10.1.16.182
2024-08-18 21:07:25,003 - __main__ - DEBUG - [client.py:53] - attempting to connect with server at ws://10.1.16.182:37020/client
2024-08-18 21:07:25,005 - __main__ - DEBUG - [client.py:133] - networkUtils.client started
2024-08-18 21:07:25,006 - __main__ - INFO - [client.py:145] - Client: Connected
2024-08-18 21:07:25,006 - __main__ - DEBUG - [client.py:157] - connected to: 10.1.16.182
2024-08-18 21:07:25,011 - __main__ - INFO - [client.py:162] - received message from: server -> (fill_buffer(self, 'server_name', 'VR1P620-05', ))
2024-08-18 21:07:25,513 - __main__ - DEBUG - [envy.py:257] - executing: await NV.fill_buffer(self, 'server_name', 'VR1P620-05', )
2024-08-18 21:09:17,145 - __main__ - INFO - [client.py:162] - received message from: server -> (restart_envy())
2024-08-18 21:09:17,276 - __main__ - DEBUG - [envy.py:257] - executing: await NV.restart_envy(self, )


2024-08-18 21:09:18.312578
21:09:182024-08-18 21:09:18,361 - __main__ - DEBUG - [envy.py:58] - choosing role
2024-08-18 21:09:18,367 - __main__ - DEBUG - [client.py:39] - health checking server
2024-08-18 21:09:18,372 - __main__ - DEBUG - [envy.py:66] - server health check status: True
2024-08-18 21:09:18,373 - __main__ - DEBUG - [envy.py:69] - I am a Client
2024-08-18 21:09:18,373 - __main__ - INFO - [envy.py:128] - Started Envy
2024-08-18 21:09:18,373 - __main__ - DEBUG - [envy.py:149] - envy.connect: Purpose is client
2024-08-18 21:09:18,373 - __main__ - DEBUG - [envy_utils.py:95] - getting server IP
2024-08-18 21:09:18,374 - __main__ - DEBUG - [envy_utils.py:107] - server file found
2024-08-18 21:09:18,375 - __main__ - DEBUG - [envy_utils.py:111] - server ip address is: 10.1.16.182
2024-08-18 21:09:18,375 - __main__ - DEBUG - [client.py:53] - attempting to connect with server at ws://10.1.16.182:37020/client
2024-08-18 21:09:18,378 - __main__ - DEBUG - [client.py:133] - networkUtils.client started
2024-08-18 21:09:18,378 - __main__ - INFO - [client.py:145] - Client: Connected
2024-08-18 21:09:18,378 - __main__ - DEBUG - [client.py:157] - connected to: 10.1.16.182
2024-08-18 21:09:18,378 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:09:18,392 - __main__ - INFO - [client.py:162] - received message from: server -> (fill_buffer(self, 'server_name', 'VR1P620-05', ))
2024-08-18 21:09:18,887 - __main__ - DEBUG - [envy.py:257] - executing: await NV.fill_buffer(self, 'server_name', 'VR1P620-05', )
2024-08-18 21:09:19,650 - __main__ - DEBUG - [client.py:171] - consumer_handler: no close frame received or sent
2024-08-18 21:09:19,861 - __main__ - WARNING - [client.py:149] - connection closed with server
2024-08-18 21:09:19,861 - __main__ - DEBUG - [client.py:188] - stopping client
2024-08-18 21:09:19,861 - __main__ - DEBUG - [client.py:98] - disconnecting websocket
2024-08-18 21:09:19,861 - __main__ - DEBUG - [client.py:106] - Connection Closed Successfully
2024-08-18 21:09:19,862 - __main__ - DEBUG - [client.py:199] - cleaning up tasks
2024-08-18 21:09:19,862 - __main__ - DEBUG - [client.py:202] - cancelling task client.consumer_handler
2024-08-18 21:09:19,862 - __main__ - DEBUG - [client.py:212] - client stopped
2024-08-18 21:09:19,862 - __main__ - DEBUG - [envy.py:198] - cleaning up envy.client_dependant_tasks
2024-08-18 21:09:19,862 - __main__ - INFO - [envy.py:214] - electing new server
2024-08-18 21:09:19,862 - __main__ - DEBUG - [envy_utils.py:149] - opening clients file at Z:/school/ENVY/__ENVYTEST__/Connections/clients.json
2024-08-18 21:09:19,876 - __main__ - DEBUG - [envy_utils.py:154] - returning clients: {'VR1P620-05': {'IP': '10.1.16.182', 'Status': 'idle', 'Job': None, 'Task': None, 'Progress': 0}, 'VR1P620-04': {'IP': '10.1.16.181', 'Status': 'idle', 'Job': None, 'Task': None, 'Progress': 0}, 'VR1P620-02': {'IP': '10.1.16.179', 'Status': 'idle', 'Job': None, 'Task': None, 'Progress': 0}}
2024-08-18 21:09:19,876 - __main__ - DEBUG - [envy.py:149] - envy.connect: Purpose is server
2024-08-18 21:09:19,876 - __main__ - DEBUG - [envy.py:106] - writing server file
2024-08-18 21:09:19,925 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:09:20,911 - __main__ - DEBUG - [envy_utils.py:95] - getting server IP
2024-08-18 21:09:20,911 - __main__ - DEBUG - [envy_utils.py:107] - server file found
2024-08-18 21:09:20,918 - __main__ - DEBUG - [envy_utils.py:111] - server ip address is: 10.1.16.182
2024-08-18 21:09:20,918 - __main__ - DEBUG - [client.py:53] - attempting to connect with server at ws://10.1.16.182:37020/client
2024-08-18 21:09:20,921 - __main__ - DEBUG - [client.py:133] - networkUtils.client started
2024-08-18 21:09:20,921 - __main__ - INFO - [client.py:145] - Client: Connected
2024-08-18 21:09:20,921 - __main__ - DEBUG - [client.py:157] - connected to: 10.1.16.182
2024-08-18 21:09:20,926 - __main__ - INFO - [client.py:162] - received message from: server -> (fill_buffer(self, 'server_name', 'VR1P620-05', ))
2024-08-18 21:09:21,365 - __main__ - DEBUG - [envy.py:257] - executing: await NV.fill_buffer(self, 'server_name', 'VR1P620-05', )
2024-08-18 21:09:36,688 - __main__ - INFO - [client.py:162] - received message from: server -> (Task: 30829973980)
2024-08-18 21:09:37,063 - __main__ - DEBUG - [envy.py:257] - executing: await NV.PLUGIN_EXAMPLE(self, '{"Purpose": "cache", "ID": 30829973980, "Environment": {"Hip": "blahblah.ma", "Job": "Z:/blahblah"}, "Parameters": {"A": 5}, "Frame": 1}', )
2024-08-18 21:09:37,114 - __main__ - INFO - [envy_plugin.py:47] - Plugin-Example_Plugin_Handler: Initialized -> Frame: 1
2024-08-18 21:09:37,122 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:09:37,345 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:37,889 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:38,308 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:38,836 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:39,379 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:39,816 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:40,361 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:40,894 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:41,328 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:41,871 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:42,403 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:42,825 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:43,370 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:43,898 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:44,321 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:44,864 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:45,391 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:45,843 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:46,392 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:46,828 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:47,377 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:47,819 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:48,348 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:48,894 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:49,346 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:49,908 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:50,344 - __main__ - DEBUG - [client.py:115] - sending message to server: (finish_task(): 30829973980)
2024-08-18 21:09:50,344 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:09:50,847 - __main__ - INFO - [client.py:162] - received message from: server -> (Task: 30829973983)
2024-08-18 21:09:51,315 - __main__ - DEBUG - [envy.py:257] - executing: await NV.PLUGIN_EXAMPLE(self, '{"Purpose": "cache", "ID": 30829973983, "Environment": {"Hip": "blahblah.ma", "Job": "Z:/blahblah"}, "Parameters": {"A": 5}, "Frame": 4}', )
2024-08-18 21:09:51,315 - __main__ - INFO - [envy_plugin.py:47] - Plugin-Example_Plugin_Handler: Initialized -> Frame: 4
2024-08-18 21:09:51,324 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:09:51,537 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:52,083 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:52,598 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:53,034 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:53,580 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:54,112 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:54,565 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:55,098 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:55,536 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:56,068 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:56,616 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:57,058 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:57,591 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:58,126 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:58,579 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:59,116 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:09:59,552 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:10:00,096 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:10:00,629 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:10:01,047 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:10:01,596 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:10:02,126 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:10:02,563 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:10:03,115 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:10:03,540 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:10:04,086 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:10:04,632 - __main__ - DEBUG - [client.py:115] - sending message to server: (finish_task(): 30829973983)
2024-08-18 21:10:04,632 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:11:05,331 - __main__ - INFO - [client.py:162] - received message from: server -> (restart_envy())
2024-08-18 21:11:05,362 - __main__ - DEBUG - [envy.py:257] - executing: await NV.restart_envy(self, )


2024-08-18 21:11:06.383957
21:11:062024-08-18 21:11:06,427 - __main__ - DEBUG - [envy.py:58] - choosing role
2024-08-18 21:11:06,432 - __main__ - DEBUG - [client.py:39] - health checking server
2024-08-18 21:11:06,438 - __main__ - DEBUG - [envy.py:66] - server health check status: True
2024-08-18 21:11:06,439 - __main__ - DEBUG - [envy.py:69] - I am a Client
2024-08-18 21:11:06,439 - __main__ - INFO - [envy.py:128] - Started Envy
2024-08-18 21:11:06,439 - __main__ - DEBUG - [envy.py:149] - envy.connect: Purpose is client
2024-08-18 21:11:06,439 - __main__ - DEBUG - [envy_utils.py:95] - getting server IP
2024-08-18 21:11:06,440 - __main__ - DEBUG - [envy_utils.py:107] - server file found
2024-08-18 21:11:06,441 - __main__ - DEBUG - [envy_utils.py:111] - server ip address is: 10.1.16.182
2024-08-18 21:11:06,442 - __main__ - DEBUG - [client.py:53] - attempting to connect with server at ws://10.1.16.182:37020/client
2024-08-18 21:11:06,445 - __main__ - DEBUG - [client.py:133] - networkUtils.client started
2024-08-18 21:11:06,445 - __main__ - INFO - [client.py:145] - Client: Connected
2024-08-18 21:11:06,445 - __main__ - DEBUG - [client.py:157] - connected to: 10.1.16.182
2024-08-18 21:11:06,445 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:11:06,459 - __main__ - INFO - [client.py:162] - received message from: server -> (fill_buffer(self, 'server_name', 'VR1P620-05', ))
2024-08-18 21:11:06,956 - __main__ - DEBUG - [envy.py:257] - executing: await NV.fill_buffer(self, 'server_name', 'VR1P620-05', )
2024-08-18 21:11:07,436 - __main__ - DEBUG - [client.py:171] - consumer_handler: no close frame received or sent
2024-08-18 21:11:07,964 - __main__ - WARNING - [client.py:149] - connection closed with server
2024-08-18 21:11:07,964 - __main__ - DEBUG - [client.py:188] - stopping client
2024-08-18 21:11:07,964 - __main__ - DEBUG - [client.py:98] - disconnecting websocket
2024-08-18 21:11:07,964 - __main__ - DEBUG - [client.py:106] - Connection Closed Successfully
2024-08-18 21:11:07,965 - __main__ - DEBUG - [client.py:199] - cleaning up tasks
2024-08-18 21:11:07,965 - __main__ - DEBUG - [client.py:202] - cancelling task client.consumer_handler
2024-08-18 21:11:07,965 - __main__ - DEBUG - [client.py:212] - client stopped
2024-08-18 21:11:07,965 - __main__ - DEBUG - [envy.py:198] - cleaning up envy.client_dependant_tasks
2024-08-18 21:11:07,965 - __main__ - INFO - [envy.py:214] - electing new server
2024-08-18 21:11:07,965 - __main__ - DEBUG - [envy_utils.py:149] - opening clients file at Z:/school/ENVY/__ENVYTEST__/Connections/clients.json
2024-08-18 21:11:07,978 - __main__ - DEBUG - [envy_utils.py:154] - returning clients: {'VR1P620-05': {'IP': '10.1.16.182', 'Status': 'idle', 'Job': None, 'Task': None, 'Progress': 0}, 'VR1P620-04': {'IP': '10.1.16.181', 'Status': 'idle', 'Job': None, 'Task': None, 'Progress': 0}, 'VR1P620-02': {'IP': '10.1.16.179', 'Status': 'idle', 'Job': None, 'Task': None, 'Progress': 0}}
2024-08-18 21:11:07,978 - __main__ - DEBUG - [envy.py:149] - envy.connect: Purpose is server
2024-08-18 21:11:07,978 - __main__ - DEBUG - [envy.py:106] - writing server file
2024-08-18 21:11:07,999 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:11:09,018 - __main__ - DEBUG - [envy_utils.py:95] - getting server IP
2024-08-18 21:11:09,019 - __main__ - DEBUG - [envy_utils.py:107] - server file found
2024-08-18 21:11:09,024 - __main__ - DEBUG - [envy_utils.py:111] - server ip address is: 10.1.16.182
2024-08-18 21:11:09,024 - __main__ - DEBUG - [client.py:53] - attempting to connect with server at ws://10.1.16.182:37020/client
2024-08-18 21:11:09,027 - __main__ - DEBUG - [client.py:133] - networkUtils.client started
2024-08-18 21:11:09,027 - __main__ - INFO - [client.py:145] - Client: Connected
2024-08-18 21:11:09,027 - __main__ - DEBUG - [client.py:157] - connected to: 10.1.16.182
2024-08-18 21:11:09,032 - __main__ - INFO - [client.py:162] - received message from: server -> (fill_buffer(self, 'server_name', 'VR1P620-05', ))
2024-08-18 21:11:09,531 - __main__ - DEBUG - [envy.py:257] - executing: await NV.fill_buffer(self, 'server_name', 'VR1P620-05', )
2024-08-18 21:11:14,729 - __main__ - INFO - [client.py:162] - received message from: server -> (Task: 88832207580)
2024-08-18 21:11:15,143 - __main__ - DEBUG - [envy.py:257] - executing: await NV.PLUGIN_EXAMPLE(self, '{"Purpose": "cache", "ID": 88832207580, "Environment": {"Hip": "blahblah.ma", "Job": "Z:/blahblah"}, "Parameters": {"A": 5}, "Frame": 1}', )
2024-08-18 21:11:15,165 - __main__ - INFO - [envy_plugin.py:47] - Plugin-Example_Plugin_Handler: Initialized -> Frame: 1
2024-08-18 21:11:15,172 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:11:15,396 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:15,831 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:16,374 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:16,895 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:17,316 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:17,861 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:18,388 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:18,823 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:19,368 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:19,901 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:20,339 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:20,883 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:21,318 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:21,866 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:22,397 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:22,833 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:23,384 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:23,817 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:24,348 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:24,892 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:25,328 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:25,860 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:26,403 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:26,839 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:27,384 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:27,820 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:28,366 - __main__ - DEBUG - [client.py:115] - sending message to server: (finish_task(): 88832207580)
2024-08-18 21:11:28,366 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:11:28,879 - __main__ - INFO - [client.py:162] - received message from: server -> (Task: 88832207583)
2024-08-18 21:11:29,327 - __main__ - DEBUG - [envy.py:257] - executing: await NV.PLUGIN_EXAMPLE(self, '{"Purpose": "cache", "ID": 88832207583, "Environment": {"Hip": "blahblah.ma", "Job": "Z:/blahblah"}, "Parameters": {"A": 5}, "Frame": 4}', )
2024-08-18 21:11:29,327 - __main__ - INFO - [envy_plugin.py:47] - Plugin-Example_Plugin_Handler: Initialized -> Frame: 4
2024-08-18 21:11:29,437 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:11:29,546 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:30,108 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:30,562 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:31,104 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:31,560 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:32,107 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:32,578 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:33,019 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:33,580 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:34,034 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:34,593 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:35,044 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:35,588 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:36,028 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:36,571 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:37,100 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:37,534 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:38,060 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:38,593 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:39,108 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:39,534 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:40,082 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:40,609 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:41,026 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:41,571 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:42,116 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:42,552 - __main__ - DEBUG - [client.py:115] - sending message to server: (finish_task(): 88832207583)
2024-08-18 21:11:42,552 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:11:42,999 - __main__ - INFO - [client.py:162] - received message from: server -> (Task: 88832207586)
2024-08-18 21:11:43,503 - __main__ - DEBUG - [envy.py:257] - executing: await NV.PLUGIN_EXAMPLE(self, '{"Purpose": "cache", "ID": 88832207586, "Environment": {"Hip": "blahblah.ma", "Job": "Z:/blahblah"}, "Parameters": {"A": 5}, "Frame": 7}', )
2024-08-18 21:11:43,503 - __main__ - INFO - [envy_plugin.py:47] - Plugin-Example_Plugin_Handler: Initialized -> Frame: 7
2024-08-18 21:11:43,534 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:11:43,740 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:44,289 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:44,724 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:45,255 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:45,783 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:46,234 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:46,762 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:47,289 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:47,706 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:48,264 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:48,794 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:49,229 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:49,774 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:11:50,212 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())


2024-08-18 21:12:31.391696
21:12:312024-08-18 21:12:31,436 - __main__ - DEBUG - [envy.py:58] - choosing role
2024-08-18 21:12:31,445 - __main__ - DEBUG - [client.py:39] - health checking server
2024-08-18 21:12:31,449 - __main__ - DEBUG - [envy.py:66] - server health check status: True
2024-08-18 21:12:31,449 - __main__ - DEBUG - [envy.py:69] - I am a Client
2024-08-18 21:12:31,449 - __main__ - INFO - [envy.py:128] - Started Envy
2024-08-18 21:12:31,449 - __main__ - DEBUG - [envy.py:149] - envy.connect: Purpose is client
2024-08-18 21:12:31,450 - __main__ - DEBUG - [envy_utils.py:95] - getting server IP
2024-08-18 21:12:31,450 - __main__ - DEBUG - [envy_utils.py:107] - server file found
2024-08-18 21:12:31,451 - __main__ - DEBUG - [envy_utils.py:111] - server ip address is: 10.1.16.179
2024-08-18 21:12:31,451 - __main__ - DEBUG - [client.py:53] - attempting to connect with server at ws://10.1.16.179:37020/client
2024-08-18 21:12:31,452 - __main__ - DEBUG - [client.py:133] - networkUtils.client started
2024-08-18 21:12:31,452 - __main__ - INFO - [client.py:145] - Client: Connected
2024-08-18 21:12:31,453 - __main__ - DEBUG - [client.py:157] - connected to: 10.1.16.179
2024-08-18 21:12:31,453 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:12:31,507 - __main__ - INFO - [client.py:162] - received message from: server -> (fill_buffer(self, 'server_name', 'VR1P620-02', ))
2024-08-18 21:12:31,952 - __main__ - DEBUG - [envy.py:257] - executing: await NV.fill_buffer(self, 'server_name', 'VR1P620-02', )
2024-08-18 21:12:32,900 - __main__ - INFO - [client.py:162] - received message from: server -> (Task: 888322075815)
2024-08-18 21:12:32,976 - __main__ - DEBUG - [envy.py:257] - executing: await NV.PLUGIN_EXAMPLE(self, '{"Purpose": "cache", "ID": 888322075815, "Environment": {"Hip": "blahblah.ma", "Job": "Z:/blahblah"}, "Parameters": {"A": 5}, "Frame": 16}', )
2024-08-18 21:12:32,995 - __main__ - INFO - [envy_plugin.py:47] - Plugin-Example_Plugin_Handler: Initialized -> Frame: 16
2024-08-18 21:12:33,086 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:12:33,195 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:33,726 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:34,261 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:34,692 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:35,240 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:35,676 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:36,221 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:36,754 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:37,175 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:37,723 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:38,253 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:38,688 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:39,249 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:39,701 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:40,245 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:40,683 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:41,226 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:41,761 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:42,199 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:42,767 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:43,220 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:43,754 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:44,205 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:44,756 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:45,199 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:45,741 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:46,274 - __main__ - DEBUG - [client.py:115] - sending message to server: (finish_task(): 888322075815)
2024-08-18 21:12:46,274 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:12:47,033 - __main__ - INFO - [client.py:162] - received message from: server -> (Task: 888322075818)
2024-08-18 21:12:47,198 - __main__ - DEBUG - [envy.py:257] - executing: await NV.PLUGIN_EXAMPLE(self, '{"Purpose": "cache", "ID": 888322075818, "Environment": {"Hip": "blahblah.ma", "Job": "Z:/blahblah"}, "Parameters": {"A": 5}, "Frame": 19}', )
2024-08-18 21:12:47,198 - __main__ - INFO - [envy_plugin.py:47] - Plugin-Example_Plugin_Handler: Initialized -> Frame: 19
2024-08-18 21:12:47,228 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:12:47,446 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:47,999 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:48,452 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:48,986 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:49,424 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:49,973 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:50,411 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:50,942 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:51,472 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:51,909 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:52,440 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:53,001 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:53,441 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:53,988 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:54,439 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:54,992 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:55,431 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:55,976 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:56,415 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:56,958 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:57,492 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:57,941 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:58,474 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:58,992 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:59,437 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:12:59,971 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:00,503 - __main__ - DEBUG - [client.py:115] - sending message to server: (finish_task(): 888322075818)
2024-08-18 21:13:00,503 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:13:01,170 - __main__ - INFO - [client.py:162] - received message from: server -> (Task: 88832207588)
2024-08-18 21:13:01,389 - __main__ - DEBUG - [envy.py:257] - executing: await NV.PLUGIN_EXAMPLE(self, '{"Purpose": "cache", "ID": 88832207588, "Environment": {"Hip": "blahblah.ma", "Job": "Z:/blahblah"}, "Parameters": {"A": 5}, "Frame": 9}', )
2024-08-18 21:13:01,389 - __main__ - INFO - [envy_plugin.py:47] - Plugin-Example_Plugin_Handler: Initialized -> Frame: 9
2024-08-18 21:13:01,498 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
2024-08-18 21:13:01,607 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:02,122 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:02,656 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:03,097 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:03,645 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:04,081 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:04,629 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:05,078 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:05,639 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:06,073 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:06,618 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:07,149 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:07,568 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:08,117 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:08,654 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:09,094 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:09,645 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:10,158 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:10,597 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:11,142 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:11,577 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:12,092 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:12,607 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:13,138 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:13,578 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:14,122 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_task_progress_to_server())
2024-08-18 21:13:14,659 - __main__ - DEBUG - [client.py:115] - sending message to server: (finish_task(): 88832207588)
2024-08-18 21:13:14,659 - __main__ - DEBUG - [client.py:115] - sending message to server: (send_status_to_server())
